src/kinoastar.h

```diff
@@ -70,15 +70,16 @@ struct cells {
 
 
 namespace KinoPlanner{
-    class KinoPathPlanner : public nav_core::BaseGlobalPlanner{
```

Comment:
this is supposed to be inherited here, i think. did it not work with out this?

Reply:
not working with this ,  I tried it is not compiling, maybe I will see again to find another possible solution

Reply:
cool I'll check too after my exam ends today.